util/nrve-niche-payment-handler.py

```diff
@@ -80,16 +82,8 @@ def sc_notify(self, event):
 
     def do_sc_notify(self, event):
 
-        event_payload = event.event_payload
```

Comment:
This went the wrong direction, as the changes here have removed even more of the code I added.